todo-app/src/components/ToggleButton/StatusToggleButton.test.js

```diff
@@ -0,0 +1,43 @@
+import React from "react";
```

Comment:
When I ran tests locally I noticed that there is currently a broken test in this file.

```
 FAIL  src/components/ToggleButton/StatusToggleButton.test.js
  <StatusToggleButton />
    ✕ toggle renders with 2 switches (188 ms)
    ✓ test click on complete, should return incomplete (22 ms)
    ✓ test click on incomplete, should return complete (21 ms)

  ● <StatusToggleButton /> › toggle renders with 2 switches

    TypeError: expect(...).toBeInTheDocument is not a function

      11 |       todoStatus={"incomplete"}
      12 |    />)
    > 13 |     expect(getByText('Done')).toBeInTheDocument();
         |                               ^
      14 |     expect(getByText('Incomplete')).toBeInTheDocument();
      15 |   })
      16 |

      at Object.<anonymous> (src/components/ToggleButton/StatusToggleButton.test.js:13:31)
```

`toBeTruthy()` could potentially be the alternative to `toBeInTheDocument()` for this situation.